.evergreen/run-tests.sh

```diff
@@ -15,6 +15,15 @@ AUTH=${AUTH:-noauth}
 MONGODB_URI=${MONGODB_URI:-}
 TEST_NPM_SCRIPT=${TEST_NPM_SCRIPT:-check:integration-coverage}
 COMPRESSOR=${COMPRESSOR:-}
+SKIP_DEPS=${SKIP_DEPS:-true}
+
+if [ "${CLIENT_ENCRYPTION}" == "true" ]; then
```

Comment:
I can revert this - but making CLIENT_ENCRYPTION required makes each task opt-into or out of encryption explicitly.  I also added this to ensure that I didn't accidentally lose FLE coverage while making changes by forgetting to set environment variables.